perf_test/blas/blas1/KokkosBlas_team_dot_perf_test.cpp

```diff
@@ -148,20 +149,20 @@ void run(int m, int repeat) {
   std::cout << "Each test input vector has a length of " << m << std::endl;
 
   // Warm up run of dot:
-  teamDotFunctor<Kokkos::View<Scalar*, MemSpace>> teamDotFunctorWarmUpInstance(
+  teamDotFunctor<Kokkos::View<Scalar*>, ExecSpace> teamDotFunctorWarmUpInstance(
```

Comment:
You will still need to give MemSpace to View's template arguments here. Using the default memory space will break on builds with Serial and Cuda both enabled for example, since CudaSpace would usually be the default memory space.

Reply:
Morning @brian-kelley -- I made the changes you recommended.  I have merged the fixes to the feature/gemv_rps_test into my gemm test.  The RPS version of the gemm test is ready to go, once we have this PR merged.